0008-flatpak-Add-a-shell-wrapper.patch

```diff
@@ -0,0 +1,25 @@
+From 0ef7d9ef5f4ce1d8c03294ea198f54a18f826e63 Mon Sep 17 00:00:00 2001
+From: Alec Leamas <leamas@nowhere.net>
+Date: Fri, 3 Aug 2018 18:50:08 +0200
+Subject: [PATCH] flatpak: Add a shell wrapper.
+
+---
+ data/opencpn.sh | 6 ++++++
+ 1 file changed, 6 insertions(+)
+ create mode 100644 data/opencpn.sh
+
+diff --git a/data/opencpn.sh b/data/opencpn.sh
+new file mode 100644
+index 00000000..a9bc4ef3
+--- /dev/null
++++ b/data/opencpn.sh
+@@ -0,0 +1,6 @@
++#!/bin/sh
```

Comment:
You can use a source of type script to generate the wrapper

Reply:
OK... but is it better? "Confused"

Reply:
It was a suggestion, I prefer having that as part of the manifest. But you're free to do whatever you prefer.